doc/classes/Resource.xml

```diff
@@ -40,8 +40,8 @@
 			<param index="0" name="subresources" type="bool" default="false" />
 			<description>
 				Duplicates this resource, returning a new resource with its [code]export[/code]ed or [constant PROPERTY_USAGE_STORAGE] properties copied from the original.
-				If [param subresources] is [code]false[/code], a shallow copy is returned; nested resources within subresources are not duplicated and are shared from the original resource. If [param subresources] is [code]true[/code], a deep copy is returned; nested subresources will be duplicated and are not shared.
-				Subresource properties with the [constant PROPERTY_USAGE_ALWAYS_DUPLICATE] flag are always duplicated even with [param subresources] set to [code]false[/code], and properties with the [constant PROPERTY_USAGE_NEVER_DUPLICATE] flag are never duplicated even with [param subresources] set to [code]true[/code].
+				If [param subresources] is [code]false[/code], a shallow copy is returned; nested resources within subresources are not duplicated and are shared with the original resource. If [param subresources] is [code]true[/code], a deep copy is returned; nested subresources will be duplicated and are not shared.
+				Subresource properties with the [constant PROPERTY_USAGE_ALWAYS_DUPLICATE] flag are always duplicated even with [param subresources] set to [code]false[/code]. Properties with the [constant PROPERTY_USAGE_NEVER_DUPLICATE] flag or subresources inside [Array] and [Dictionary] properties are never duplicated, not even with [param subresources] set to [code]true[/code].
```

Comment:
Not sure with the extra "not" towards the end, but I'm not a native speaker.
This line is specifically talking about **PROPERTY_USAGE_ALWAYS_DUPLICATE** and **PROPERTY_USAGE_NEVER_DUPLICATE**. I would be careful by not by extending the same sentence too much, as it's a fairly unorthodox topic already.
subresources inside [Array] and [Dictionary] properties 
```suggestion
				Subresource properties with the [constant PROPERTY_USAGE_ALWAYS_DUPLICATE] flag are always duplicated even with [param subresources] set to [code]false[/code]. Properties with the [constant PROPERTY_USAGE_NEVER_DUPLICATE] flag are never duplicated, not even with [param subresources] set to [code]true[/code]. Subresources inside [Array] and [Dictionary] properties are also [b]never[/b] duplicated.
```

Reply:
@Mickeon what do you think of the latest, where I opted for a list of the three exceptions?

I've been looking for a way not to have to say `[param subresources] set to [code]true[/code]` three times.